civihr_employee_portal/updates/7043.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Remove Vacancies feature from the slideshow after new contact onboarding
+ */
+function civihr_employee_portal_update_7043() {
+  $vacanciesNodeTitle = 'Vacancies';
+  $vacanciesNodeId = db_query("SELECT nid FROM node WHERE title = '{$vacanciesNodeTitle}'")
+    ->fetchField();
+
+  $vacanciesNodeId && node_delete($vacanciesNodeId);
```

Comment:
While the shorthand looks good here, It's not easy to catch what is happening at first glance, for more clarity I would suggest we go the conditional route.

```php
if ($vacanciesNodeId) {
node_delete($vacanciesNodeId)
}